cloud_governance/zombie_cluster/delete_ec2_resources.py

```diff
@@ -38,6 +36,7 @@ class DeleteEC2Resources:
           Security Groups, NetworkACl]
     """
 
+    @typeguard.typechecked
```

Comment:
I dont think we need typecheck to init method

Reply:
ok, sir. I will remove it.